make-release.sh

```diff
@@ -56,7 +59,10 @@ else
     # change back manifests to :latest
     find "${REPO_ROOT}/kubernetes-manifests" -name '*.yaml' -exec sed -i -e "s/:${NEW_VERSION}/:latest/g" {} \;
     git add "${REPO_ROOT}/kubernetes-manifests/*.yaml"
-    git commit -m "revert to latest images"
+    git commit -m "revert demo app images to latest"
+    find "${REPO_ROOT}/kubernetes-config" -name '*.yaml' -exec sed -i -e "s/:${NEW_VERSION}/:latest/g" {} \;
+    git add "${REPO_ROOT}/kubernetes-config/*.yaml"
+    git commit -m "revert loadgeenerator images to latest"
```

Comment:
Also, this is a larger problem I'm just noticing now: Why is the load generator in a new `/kubernetes-config` file? It's confusing to have both that and `/kubernetes-manifests` in the repo root. i'd suggest putting your manifests in `/kubernetes-manifests/load-generator`